index.bs

```diff
@@ -5630,13 +5630,13 @@ The <dfn export for=commands>emulation.setGeolocationOverride</dfn> command modi
       }
 
       emulation.GeolocationCoordinates = {
-         latitude: float,
-         longitude: float,
-         ? accuracy: float .default 1.0,
+         latitude: -90.0..90.0,
```

Comment:
I can see that we have such definitions already in the spec but it's a bit sad that we loose the actual type definition here by using dots. Would the following work as well?

```suggestion
         latitude: float .ge -90.0 .le 90.0
```

Reply:
Probably we could, but I'm not sure what exactly is the problem. We have numbers with decimal places in the range, this implies that the value should be of type float.

Reply:
So we have both variants in use in the specification. It's not a blocker for this PR but maybe we should find a solution to use one or the other.

Reply:
I filed https://github.com/w3c/webdriver-bidi/issues/907 for it.